web/src/views/SubmissionPortal/Components/MultiOmicsDataForm.vue

```diff
@@ -268,15 +277,15 @@ export default defineComponent({
         </div>
 
         <v-radio
-          :value="multiOmicsForm.otherAward"
+          :value="OTHER"
```

Comment:
Is it necessary to make this use a variable? I think I would set inline with a string instead:
```suggestion
          value="OTHER"
```
This might just come down to preference but I don't typically use a component scoped const in this case.

Reply:
I did it that way because the function in `otherAwardValidationRules` checks against the value the `OTHER` constant. Without the variable there would be two hardcoded `"OTHER"` strings floating around; not the end of the world, but the variable made more sense to me.

Reply:
Fair enough. Definitely fine to keep in.